platforms/php/versions/buster/versionsToBuild.txt

```diff
@@ -60,4 +60,4 @@
 8.2.21, 8cc44d51bb2506399ec176f70fe110f0c9e1f7d852a5303a2cd1403402199707, 1198C0117593497A5EC5C199286AF1F9897469DC 39B641343D8C104B2B146DC3F9C39DC0B9698544,
```

Comment:
should we remove these files for buster/focal/stretch
<img width="922" alt="image" src="https://github.com/user-attachments/assets/d16bea7c-7eff-4320-9bfa-5a08e915ea52">